examples/automm/production/hf_to_onnx_example.py

```diff
@@ -0,0 +1,294 @@
+from autogluon.multimodal import MultiModalPredictor
+
+import argparse
+import numpy as np
+import onnx
+import onnxruntime as ort
+import torch
+from datasets import load_dataset
+from torch import tensor
+
+
+VALID_HF_INPUT = [
+    "hf_text_text_token_ids",
+    "hf_text_text_valid_length",
+    "hf_text_text_segment_ids",
+]
+
+VALID_ONNX_INPUT = [
+    "hf_text_text_token_ids",
+    "hf_text_text_valid_length",
+    "hf_text_text_segment_ids",  # comment this line for mpnet
+]
+
+
+def get_hf_model(predictor, device="cpu"):
+    model = predictor._model
+    model.to(device)
+    model.eval()
+    return model
+
+
+def export_to_onnx(model, batch=None, onnx_path="dummy.onnx", verbose=False):
```

Comment:
Do we this function? MultimodalPredictor already has the API.

Reply:
Removed the whole file and created a new example.